packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/features/exposurepoint/ExposurePointFeature.java

```diff
@@ -78,9 +81,13 @@ public void updateBuilder(@NonNull CaptureRequest.Builder requestBuilder) {
     if (!checkIsSupported()) {
       return;
     }
-    requestBuilder.set(
-        CaptureRequest.CONTROL_AE_REGIONS,
-        exposureRectangle == null ? null : new MeteringRectangle[] {exposureRectangle});
+
+    if (exposureRectangle != null) {
+      requestBuilder.set(
+          CaptureRequest.CONTROL_AE_REGIONS, new MeteringRectangle[] {exposureRectangle});
+    } else if (shouldReset(requestBuilder)) {
+      requestBuilder.set(CaptureRequest.CONTROL_AE_REGIONS, defaultExposureRectangle);
```

Comment:
I am suggesting that instead of this `else if` clause, you use an `else` and use a try/catch within this clause to try setting it to null. That removes the need for a `shouldReset` check and assuming a default value while also avoids the crash with a helpful error that can be handled by Flutter apps.

Reply:
I removed unnecessary codes/tests. I have now made the recommended changes and added a new test.